install.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+DIRECTORY="fcc-python-vagrant"
+
+echo "This script requires superuser access to install apt packages."
+echo "You will be prompted for your password by sudo."
+
+# clear any previous sudo permission
+sudo -k
+
+# run inside sudo
+sudo sh <<SCRIPT
+
+echo "Installing VirtualBox ..."
+
+if hash virtualbox 2>/dev/null; then
+	echo "Virtualbox already installed, skipping ..."
+else
+	wget -q https://www.virtualbox.org/download/oracle_vbox_2016.asc -O- | apt-key add -
+	wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | apt-key add -
+
+	apt-get update -y
+	apt-get install -y virtualbox
+fi
+
+echo "Installing dkms ..."
+if hash dkms 2>/dev/null; then
+	echo "Dkms already installed, skipping ..."
+else
+	apt-get install -y dkms
+fi
+
+echo "Installing VirtualBox extensions ..."
```

Comment:
I have it installed, but it still downloaded about 80MB of stuff. See if you can skip this too, if already at the latest, otherwise uninstall-install.